### PR TITLE
[Dictionary] mobilePickDate to movieTouched

### PR DESCRIPTION
**Pull request by livecodealice**
mobilePickDate (command): Adjusted linespacing to dispose of "One of the following items:"
mobileSensorReading (function): reformatted returns details.
mobileSetFullscreenRectForOrientation (command): Added missing reference.
label (property): Fixed broken link.
mouseChunk (function): Fixed broken link.
mouseControl (function): Removed reference to non-existent entry.
mouseDoubleDown (message): Corrected a reference's type.
mouseDoubleUp (message): Corrected a reference's type.
mouseUp (message): Fixed broken links
mouseUpInBackdrop (message): Changed punctuation to make it stop looking like text was missing.
HTMLText (property): Removed formatting that separated a word from the rest of the sentence it was in.
movieTouched (message): Replaced incorrect reference.

### CODE REVIEW

File: docs/dictionary/message/movieTouched.lcdoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ Platforms: mobile
 
 Example:
 on movieTouched
-  play stop
```
Comment (livecodepanos) [inline docs/dictionary/message/movieTouched.lcdoc]: @livecodesam the command `play stop` is valid, so these 3 changes need to be reverted.

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 968b821903dbae863435444eff5bd52d1d7a3979

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 968b821903dbae863435444eff5bd52d1d7a3979

**Comment by livecode-vulcan** [comment]
😎 test success 968b821903dbae863435444eff5bd52d1d7a3979

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/610)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/19)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/19)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/19)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/19)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2136)
* try-community-universal-ios-iphoneos12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.1/builds/361)
* try-community-universal-ios-iphonesimulator12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.1/builds/357)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2165)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2152)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2151)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2185)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2125)

**Pull request merged**